Makefile

```diff
@@ -132,11 +145,44 @@ else
 	pip install -r requirements-tests.txt
 endif
 
-
 test:   lint
 	@ $(ENTER_VENV) \
 	pytest -vv --cov-report term-missing --cov=leapp tests/scripts
 
+test_container:
+	@case $(_TEST_CONTAINER) in \
+		rhel7) \
+			export _VENV=python2.7; \
+			export _CONT_FILE="res/container-tests/Containerfile.ubi7"; \
+			;; \
+		rhel8) \
+			export _VENV=python3.6; \
+			export _CONT_FILE="res/container-tests/Containerfile.ubi8"; \
+			;; \
+		rhel9) \
+			export _VENV=python3.9; \
+			export _CONT_FILE="res/container-tests/Containerfile.ubi9"; \
+			;; \
+		*) \
+			echo "Available test containers are rhel7, rhel8, rhel9"; \
+			exit 1; \
+			;; \
+	esac; \
+	export TEST_IMAGE="leapp-tests-$(_TEST_CONTAINER)"; \
+	rm -rf testenv/ && \
+	podman build -t $$TEST_IMAGE --env PYTHON_VENV=$$_VENV -f $$_CONT_FILE res/container-tests && \
```

Comment:
`podman build` does not know the `--env` nor `-e` option. at least the one I am using

```
$ podman version
Version:      3.4.7
API Version:  3.4.7
Go Version:   go1.16.15
Built:        Mon Jul 11 00:35:25 2022
```

Reply:
It is indeed a version thing. Seems like the option is not needed at all.